Functions.Templates/ProjectTemplate/CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,55 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Functions",
+        "Solution"
+    ],
+    "name": "Azure Functions",
+    "generatorVersions": "[1.0.0.0-*)",
+    "groupIdentity": "Microsoft.AzureFunctions.ProjectTemplates",
+    "precedence": "100",
+    "identity": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.2.x",
```

Comment:
Yeah we want this to be 3.x to work in VS Code

There's an [old issue](https://github.com/Azure/azure-functions-templates/issues/916) about whether or not _item_ templates should use 3.x (and we should probably keep that a separate discussion), but the _project_ template has been 3.x for a while even in non-isolated.

Reply:
It should be 3.x. Should we do the same for item templates as well? For item templates i believe we had to revert to 2.x because VS was throwing null reference exception.

Reply:
Yeah I think I may accidentally used the `dev` branch value instead of `v3.x`. I will update this Project Template value to be `3.x` just to align with what we have. I will leave item templates as is, and we can take that issue separately if we do that across .NET Core 3 as well.

Reply:
We cannot do that for .Net Core 3 because of legacy code in old VS versions. In fact we did that and we had to revert it because of issues. If we do that here too then we would be stuck with 2.x in item templates for .Net 5 worker. Also there is no reason for us not use 3.x in item templates given that they are completely new and in a separate package.